modules/virt-installing-virtio-drivers-installing-windows.adoc

```diff
@@ -1,7 +1,7 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc
-// * virt/virtual_machines/virt-installing-qemu-guest-agent.adoc
+// * virt/backup_restore/virt-managing-vm-snapshots.adoc
+// * virt/virtual_machines/creating_vms/virt-installing-qemu-guest-agent.adoc
```

Comment:
I know it is existing text, but having a [single substep under Step 2](https://github.com/openshift/openshift-docs/pull/64685/files#diff-53f3fbc1bb09feb44e35ce616067340425610e73af23188f349e162888238618R27-R28) looks weird.